kotlin/open-mobile-map-package/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,6 @@
+<resources>
+    <string name="app_name">Open mobile map package</string>
+    <string name="config_data_sdcard_offline_dir">ArcGIS/samples/MapPackage</string>
```

Comment:
since we're moving towards using scoped storage, I don't think there's much point in creating a folder structure within the 'files' folder